### ISSUE TITLE
Reserve Support

### ISSUE THREAD

**Issue by creatorfromhell** [Realizedd/TokenManager#35]
Hello, I'm the developer of Reserve, which aims at bringing a modern Economy API to the Bukkit API. I'm currently reaching out to various Economy Plugin developers to get it implemented into them, and start branching out thirdy-party implementations. We currently offer a classic API, Async methods, and an extended API that uses the defined currency, and account objects. Please let me know if you need help implementing it.

Link: https://github.com/TheNewEconomy/Reserve

Advantages over Vault:
- Multi-currency support
- Async support using Java's completeable futures
- Optional Extended API to allow third party plugins to add new currencies

Thanks,
creatorfromhell

**Comment by Realizedd** [comment]
That's quite a lot to implement even for the classic, but I'll definitely look into it in the future.

**Comment by creatorfromhell** [comment]
Yes, its a hefty amount of features but if you dont have multi currency support 80 % of them can just be redirected to another method
